web/src/app/admin/connectors/[connector]/pages/DynamicConnectorCreationForm.tsx

```diff
@@ -10,19 +10,24 @@ import { TextFormField } from "@/components/admin/connectors/Field";
 import ListInput from "./ConnectorInput/ListInput";
 import FileInput from "./ConnectorInput/FileInput";
 import { AdvancedOptionsToggle } from "@/components/AdvancedOptionsToggle";
+import { AccessTypeForm } from "@/components/admin/connectors/AccessTypeForm";
+import { AccessTypeGroupSelector } from "@/components/admin/connectors/AccessTypeGroupSelector";
+import { ConfigurableSources } from "@/lib/types";
 
 export interface DynamicConnectionFormProps {
   config: ConnectionConfiguration;
   selectedFiles: File[];
   setSelectedFiles: Dispatch<SetStateAction<File[]>>;
   values: any;
+  connector: string;
```

Comment:
nit: would prefer to enforce type `ConfigurableSources` up here

Reply:
agreed!